interface/patient_file/summary/demographics.php

```diff
@@ -191,1287 +192,1009 @@ function image_widget($doc_id, $doc_catg)
+                                    'address' => [
+                                        'line1' => $adobj->get_line1(),
+                                        'line2' => $adobj->get_line2(),
+                                        'city' => $adobj->get_city(),
+                                        'state' => $adobj->get_state(),
+                                        'postal' => $adobj->get_zip(),
+                                        'country' => $adobj->get_country()
+                                    ],
+                                ];
+                                $row['policy_type'] = (!empty($row['policy_type'])) ? $policy_types[$row['policy_type']] : false;
+                            }
+                            $row['dispFromDate'] = ((strcmp($row['date'], '0000-00-00') != 0) || $row['date'] == "") ? true : false;
+                            $mname = ($row['subscriber_mname'] != "") ? $row['subscriber_mname'] : "";
+                            $row['subscriber_full_name'] = str_replace("%mname%", $mname, "{$row['subscriber_fname']} %mname% {$row['subscriber_lname']}");
+                            $insArr[] = $row;
+                        }
 
-            $fixedWidth = false;
-            expand_collapse_widget(
-                $widgetTitle,
-                $widgetLabel,
-                $widgetButtonLabel,
-                $widgetButtonLink,
-                $widgetButtonClass,
-                $linkMethod,
-                $bodyClass,
-                $widgetAuth,
-                $fixedWidth
-            );
-            ?>
-            <br />
-            <div class="text ml-2"><div class="spinner-border spinner-border-sm" role="status"><span class="sr-only"><?php echo xlt("Loading"); ?>...</span></div></div>
-            <br />
-          </div>
-        </section>
-<?php } // end labs authorized ?>
-
-<?php if ($vitals_is_registered && AclMain::aclCheckCore('patients', 'med')) { ?>
-    <section>
-            <?php // vitals expand collapse widget
-            $widgetTitle = xl("Vitals");
-            $widgetLabel = "vitals";
-            $widgetButtonLabel = xl("Trend");
-            $widgetButtonLink = "../encounter/trend_form.php?formname=vitals";
-            $widgetButtonClass = "";
-            $linkMethod = "html";
-            $bodyClass = "notab";
-            // check to see if any vitals exist
-            $existVitals = sqlQuery("SELECT * FROM form_vitals WHERE pid=?", array($pid));
-            if ($existVitals) {
-                $widgetAuth = true;
-            } else {
-                $widgetAuth = false;
-            }
+                        if ($GLOBALS["enable_oa"]) {
+                            if ($_POST['status_update'] === 'true') {
+                                unset($_POST['status_update']);
+                                $showEligibility = true;
+                                $ok = EDI270::requestEligibility($pid);
```

Comment:
this seems like an odd change (is this demographics.php script outdated?)

Reply:
I sure hope it's not outdated... That would be a tremendous amount of work for naught. This isn't technically an addition, this code just got moved around.

Reply:
I just checked, this code exists on master, script is up to date

Reply:
![Uploading 2DV-2.gif…]()